db/seqno_to_time_mapping.h

```diff
@@ -18,20 +18,32 @@
 
 namespace ROCKSDB_NAMESPACE {
 
-constexpr uint64_t kUnknownSeqnoTime = 0;
-
-// SeqnoToTimeMapping stores the sequence number to time mapping, so given a
-// sequence number it can estimate the oldest possible time for that sequence
-// number. For example:
-//   10 -> 100
-//   50 -> 300
-// then if a key has seqno 19, the OldestApproximateTime would be 100, for 51 it
-// would be 300.
-// As it's a sorted list, the new entry is inserted from the back. The old data
-// will be popped from the front if they're no longer used.
+constexpr uint64_t kUnknownTimeBeforeAll = 0;
+constexpr SequenceNumber kUnknownSeqnoBeforeAll = 0;
+
+// SeqnoToTimeMapping stores a sampled mapping from sequence numbers to
+// unix times (seconds since epoch). This information provides rough bounds
+// between sequence numbers and their write times, but is primarily designed
+// for getting a best lower bound on the sequence number of data written no
```

Comment:
If we record 10 -> 100 and 20 -> 200 and want to the sequence number at time 150, we essentially have [10, 20] as our best range of possibilities, from which we would use the lower bound 10. "Best" because 0 is also a valid lower bound.

Reply:
That makes sense, thanks.